opentracing/src/main/java/opentracing/TraceCodec.java

```diff
@@ -24,16 +24,16 @@
  * binary message format, so that it can be inspected efficiently by the middleware / routing layers
  * without parsing the whole message.
  */
-public interface TraceContextCodec {
+public interface TraceCodec {
```

Comment:
> how else would a framework or an application serialize the context into whichever protocol they are using for RPC?

Any way they like.

The specification explains the need for serialisation.
But I don't (yet) see the need for the API to impose how it should be done.